app/api/routes.py

```
@@ -226,6 +226,30 @@ def get_record_exchange_by_id(
+def delete_record_route(
+        rec_id: int,
+        db: Session = Depends(get_db),
+        current_user: User = Depends(AuthTokenDep)
+):
+    try:
+        delete_record(db, rec_id, current_user.id)
+        return Response(status_code=204)
+
+    except HTTPException:
+        raise
+    except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)